### DISCUSSION TITLE
Light mode

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [daya0576/beaverhabits#34] [Ideas]
Could we have a light mode too, please?

**Comment by daya0576** [comment]
Thanks for the requirement, that's definitely something that should be provided.

It will require some effort to redesign and remove all anti-pattern code, but I will inform you once this feature becomes available.

**Comment by daya0576** [comment]
Hi, light mode is finally available with https://github.com/daya0576/beaverhabits/pull/125. You can take a preview at the demo page: https://beaverhabits.com/demo

**Reply by daya0576** [reply]
A new release is coming soon:   
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/9ef8f1f2-d6f2-489c-a36d-3fb6e23ba83e" />
